CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Updated
+
+-Updated Pod Eviction Method for Drain Node Scenarios
+
```

Comment:
Good catch to add to the changelog. It's often overlooked.

Please simply state you updated to API/v1 as per the lifecycle of Kubernetes API which states the pod eviction has been V1beta1Eviction was deprecated and removed in Kubernetes 1.22.